LoopProjectFile/Extents.py

```diff
@@ -1,6 +1,9 @@
 # import netCDF4
 
 
+from pyproj import CRS
```

Comment:
I'm not sure if its a good idea to require pyproj, it means we have another library with gdal as a dependency. @RoyThomsonMonash do you have any comment?

Reply:
Avoiding importing pyproj is a good idea as you say, the conversion between utm zones and epsg is very simple so is also not needed in this case.